assets/js/sponsorshipLevelBenefit/edit.js

```diff
@@ -0,0 +1,209 @@
+'use strict';
+
+import $ from 'jquery';
+import Routing from '../Components/Routing';
+import '../../css/sponsorshipLevelBenefit/edit.scss';
+
+$(document).ready(function() {
+    updateLevelArrows();
+    updateBenefitArrows();
+    initSponsorshipLevelBenefitActions();
+});
+
+/* edit position of Sponsorhip Level */
+function sendLevelPosition(level_id, move)
+{
+    let token = $('#sponsorshipLevelBenefitForm').find('input[name="_token"]').val();
+
+    $.ajax({
+        url: Routing.generate('sponsorship_level_benefit_edit_level_position'),
+        type: "PUT",
+        DataType: "json",
+        data: {
+            id: level_id,
+            move: move,
+            _token: token
+        },
+        success: function(result) {
+            moveLevel(level_id, result['new_level_id'], move);
+        }
+    });
+}
+
+function moveLevel(old_id, new_id, move)
+{
+    $('.benefit_tr').each(function() {
+        let old_content = '<td data-id="'+old_id+'">' + $(this).find('td[data-id="'+old_id+'"]').html() + '</td>';
+        $(this).find('td[data-id="'+old_id+'"]').remove();
+        if (move === 'left') {
+            $(this).find('td[data-id="'+new_id+'"]').before(old_content);
+        }
+        else {
+            $(this).find('td[data-id="'+new_id+'"]').after(old_content);
+        }
+    });
+
+    let old_content = '<th class="level_th" data-id="'+old_id+'">' + $('.level_th[data-id="'+old_id+'"]').html() + '</th>';
+    $('.level_th[data-id="'+old_id+'"]').remove();
+    if (move === 'left') {
+        $('.level_th[data-id="'+new_id+'"]').before(old_content);
+    }
+    else {
```

Comment:
Remove this else, either by composing before/after using a variable and then invoke the method with it, or using another strategy